repos/spack_repo/builtin/packages/paraview/package.py

```diff
@@ -152,18 +152,31 @@ class Paraview(CMakePackage, CudaPackage, ROCmPackage):
         ' "default" lets the build_edition make the decision.'
         ' "on" or "off" will always override the build_edition.',
     )
+    variant(
+        "use_viskores",
+        default="default",
+        multi=False,
+        values=("default", "on", "off"),
+        description="Build Viskores with ParaView by setting PARAVIEW_USE_VISKORES=ON,OFF."
```

Comment:
Here and `when="@:5"` for the `use_vtkm` variant. This should make it possible to drop the added version constraint everywhere these are used.

```suggestion
        when="@6:",
        description="Build Viskores with ParaView by setting PARAVIEW_USE_VISKORES=ON,OFF."
```